community/io/src/main/java/org/neo4j/io/pagecache/impl/muninn/PageList.java

```diff
@@ -535,4 +535,9 @@ void toString( long pageRef, StringBuilder sb )
         sb.append( ", usageCounter = " ).append( getUsageCounter( pageRef ) );
         sb.append( " ] " ).append( OffHeapPageLock.toString( offLock( pageRef ) ) );
     }
+
+    public void close()
+    {
+        memoryAllocator.close();
+    }
```

Comment:
`PageList` do not have ownership of the `MemoryAllocator`, so I don't think they should be allowed to do this. A secondary confusing effect is also that `MuninnPagedFile` extends `PageList` and already has a `close` method, which now overrides this method. We definitely do not intend for this to mean that the closing of one paged file implies the deallocation of the entire page cache.